goals.md

```diff
@@ -9,29 +9,29 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## 2022
+## [PROPOSED] 2022
 
-### 1. Cultivate growth and adoption of OSPOs
+### 1 Cultivate growth and adoption of OSPOs
 
-* Release at least 6 new OSPO case studies
-* Launch 6 new OSPO guides
-* Host 12 OSPOlogy sessions
-* Build out and run OSPO Survey 2022
-* Create and publish a OSPO Maturity Model
+- Release at least 6 new OSPO case studies
+- Launch 6 new OSPO guides
+- Host 12 OSPOlogy sessions
+- Build out and run OSPO Survey 2022
+- Create and publish a OSPO Maturity Model
 
-## 2. Grow the membership base
+## 2 Grow the membership base
 
-* Grow TODO membership by 20%
-* Cultivate relationships with other OSPO adjacent communities like CHAOSS
-* Grow OSPO Ambassadors by 
+- Grow TODO membership by 20%
+- Cultivate relationships with other OSPO adjacent communities like CHAOSS
+- Grow OSPO Ambassadors by 
 
-## 3. Create value for all members
+## 3 Create value for all members
```

Comment:
Re: @itprofjacobs & @tsteenbe's comments & related discussion at the 2022-01-11 call, we need to have a closer look at what "all members" means here. There may be different cohorts depending upon experience & maturity. The TODO Group value proposition for early stage OSPOs will be different than for OSPOs who have been around for a while.